modules/doctor/mergebase.go

```diff
@@ -30,6 +30,7 @@ func iteratePRs(ctx context.Context, repo *repo_model.Repository, each func(*rep
 }
 
 func checkPRMergeBase(ctx context.Context, logger log.Logger, autofix bool) error {
+	git.InitOnceWithSync(ctx)
```

Comment:
Is this check broken?

Reply:
yes it will be since the move to have the git init function